### ISSUE TITLE
Missing strip() causes inconsistent results

### ISSUE THREAD

**Issue by zuny26** [bitextor/bicleaner#46]
If I want to run Bicleaner on some data that has columns in a different order than my model, as far as I understand, I can either:
* Use `--scol` and `--tcol` parameters to indicate where source and target are
* Change the column order in the data to match the expected values (`scol`=3, `tcol`=4)

It seems like these options should give the same result, but they don't. In the example I provide, I have `url1 url2 text_ca text_es` columns, and I am using the `es-ca` model from the latest [release](https://github.com/bitextor/bicleaner-data/releases/tag/v1.4). 
[bicleaner_input.txt](https://github.com/bitextor/bicleaner/files/5223395/bicleaner_input.txt)

Running Bicleaner with these commands gives mildly different results:
```
$ cat bicleaner_input.txt \
  | python3 bicleaner_classifier_lite.py --score_only --scol 4 --tcol 3 - - es-ca/es-ca.yaml
0.683
0.863
0.883
```

```
$ cat bicleaner_input.txt \
  | awk -F $'\t' '{print $2,$1,$4,$3}' OFS=$'\t' \
  | python3 bicleaner_classifier_lite.py --score_only - - es-ca/es-ca.yaml
0.687
0.880
0.890
```

I think this comes from this line: https://github.com/bitextor/bicleaner/blob/9e9937cdf26b50b86fd76efa7e9d9dd28c891297/bicleaner/bicleaner_classifier_lite.py#L231 Since there is no `strip()` expression, one of the sentences will have a new line, which will be on the source side when using `--scol` and `--tcol`, and on the target side if the columns are reordered beforehand. So, obviously, this only happens when the sentences are the last columns of the input.

**Comment by mbanon** [comment]
Thanks for the report, @zuny26 . We'll take a look into it :)

**Comment by mbanon** [comment]
Hi @zuny26 , 
I fixed it in brach `bicleaner-0.15`   https://github.com/bitextor/bicleaner/commit/d504a62135f4b247e9177c7016fdfb28a27e0858
